common/protob/messages-cardano.proto

```diff
@@ -223,8 +223,17 @@ message CardanoSignTx {
 /**
  * Response: Serialised signed cardano transaction
  * @end
+ * @next CardanoSignedTxAck
  */
 message CardanoSignedTx {
-    required bytes tx_hash = 1;         // hash of the transaction body
-    required bytes serialized_tx = 2;   // serialized, signed transaction
+    required bytes tx_hash = 1;               // hash of the transaction body
+    required bytes serialized_tx = 2;         // serialized, signed transaction
+    optional bool expect_more_chunks = 3;     // whether more chunks of the transaction follow
+}
+
+/**
+ * Request: Serialised signed cardano transaction acknowledgement
+ * @next CardanoSignedTx
+ */
+message CardanoSignedTxAck {
```

Comment:
Wouldn't better name be SignedTxCont(inuation) ? Or maybe I am misrepresenting the meaning of the message :shrug:

Reply:
this is the message sent by the client back to the trezor device signalizing the client processed the chunk of the serialized transaction and is ready to receive the next one so it imho makes sense to call it an acknowledgement.

Analogous naming can be found in e.g. in Ethereum: https://github.com/trezor/trezor-firmware/blob/17324a01a3b71aa79ab96b95a252488ba2701bb7/common/protob/messages-ethereum.proto#L90 or